include/roboteam_observer/Handler.h

```diff
@@ -0,0 +1,39 @@
+#ifndef WORLDHANDLER_H
+#define WORLDHANDLER_H
+
+#include <roboteam_proto/RobotData.pb.h>
+#include <roboteam_proto/State.pb.h>
+#include <net/robocup_ssl_client.h>
+#include <Publisher.h> //TODO: ugly include, fix in utils
+#include <Subscriber.h>
+#include <observer/Observer.h>
+#include <utility>
+
+
+
+class Handler {
+   private:
+    std::unique_ptr<proto::Publisher<proto::State>> pub_state = nullptr;
+    std::unique_ptr<RoboCupSSLClient> vision_client = nullptr;
+    std::unique_ptr<RoboCupSSLClient> referee_client = nullptr;
+    std::unique_ptr<proto::Subscriber<proto::RobotData>> sub_feedback = nullptr;
+    std::unique_ptr<proto::Subscriber<proto::RobotData>> sub_feedback_2 = nullptr;
```

Comment:
99% confident these don't have to be heap allocated.

Reply:
I don't like that, personally. Now it's easier to control when networking connections are started/destroyed. It's a marginal performance difference at best anyways.

Reply:
Fair enough, C++ initialization can be hard sometimes :P